helix-core/src/test/java/org/apache/helix/integration/rebalancer/TestAutoRebalanceWithDisabledInstance.java

```diff
@@ -43,6 +45,7 @@ public void beforeClass() throws Exception {
     super.beforeClass();
     _gSetupTool.addResourceToCluster(CLUSTER_NAME, TEST_DB_2, _PARTITIONS, STATE_MODEL,
         RebalanceMode.FULL_AUTO.name(), CrushEdRebalanceStrategy.class.getName());
+    setupAutoRebalancer();
```

Comment:
Again add the comment that we need autoRebalancer + autoRebalance strategy as this is autoRebalance test